conf.yaml

```diff
@@ -1075,6 +1076,21 @@ contents:
           - title:      Enterprise Search Clients
             base_dir:   en/enterprise-search-clients
             sections:
+              - title:      Enterprise Search PHP client
+                prefix:     php
+                current:    7.16
+                branches:   [ 7.16 ]
+                live:       [ 7.16 ]
```

Comment:
`live` isn't needed if it's the same as `branches`, but it's probably good to leave this in for now as they'll diverge in the future.